streams/jvm/src/main/scala/scalaz/zio/stream/platform.scala

```diff
@@ -0,0 +1,51 @@
+package scalaz.zio.stream
+
+import java.io.{ InputStream, OutputStream }
+
+import scalaz.zio._
+import scalaz.zio.blocking._
+
+trait ZStreamPlatformSpecific {
+
+  /**
+   * Uses the provided `TaskR` value to create a [[ZStream]] of byte chunks, backed by
+   * the resulting `InputStream`. When data from the `InputStream` is exhausted,
+   * the stream will close it.
+   */
+  def fromInputStream[R](
+    is: TaskR[R, InputStream],
+    chunkSize: Int = 4096
```

Comment:
We should probably put the constant in `ZStreamChunk.DefaultChunkSize` so we can easily change it across all code.

Reply:
good idea